cmake/Findlibmaxminddb.cmake

```diff
@@ -0,0 +1,53 @@
+# Findlibmaxminddb
+# ---------
+#
+# Find the libmaxminddb includes and library.
+#
+# Result Variables
+# ^^^^^^^^^^^^^^^^
+#
+# This module will set the following variables in your project:
+#
+# ``libmaxminddb_INCLUDE_DIRS``
+#   where to find maxminddb.h, etc.
+# ``libmaxminddb_LIBRARIES``
+#   the libraries to link against to use the MaxMindDB library.
+# ``libmaxminddb_VERSION``
+#   the version of the MaxMindDB library found.
+# ``libmaxminddb_FOUND``
+#   true if the MaxMindDB library was found.
+#
```

Comment:
Where is this file from?

Reply:
I copied it from `Findlibunwind.cmake`.

Reply:
Okay, then forget this—I thought it was copied from an external source.